apps/services/auth/ids-api/src/app/cleanup/cleanup.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Inject } from '@nestjs/common'
+
+import { GrantsService } from '@island.is/auth-api-lib'
+import { LOGGER_PROVIDER } from '@island.is/logging'
+
+import type { Logger } from '@island.is/logging'
+export class CleanupService {
```

Comment:
~ Lint formatting should add a line between imports and code.
```suggestion
import type { Logger } from '@island.is/logging'

export class CleanupService {
```